st2common/bin/st2-track-result

```diff
@@ -52,10 +52,6 @@ def add_result_tracker(exec_id):
     # Check runner type.
```

Comment:
If results tracker was only supported for mistral, then should this whole file go? Or should we leave it in with the runner_type != mistral-v2 so the logic is there if need to add for new runners?
What about the actual results tracker itself?